joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -142,14 +142,16 @@ trait AstCreatorHelper(implicit withSchemaValidation: ValidationMode) { this: As
       "class"
     )
 
-  private val KeepTypeKeywords: List[String] = List("unsigned", "volatile")
+  private val KeepTypeKeywords: List[String] = List("unsigned", "volatile", "const", "static")
```

Comment:
I'd rename this, since `static` is decidedly not part of the type. (I think it's called a storage modifier.)

Reply:
Well, it's a list of keywords to keep at the string representation of a type. I keep it that way for now.